Library/Subscription.cs

```diff
@@ -801,10 +815,15 @@ protected void WriteChangeSubscriptionXml(XmlTextWriter xmlWriter, ChangeTimefra
             xmlWriter.WriteStartElement("subscription"); // Start: subscription
 
             xmlWriter.WriteElementString("timeframe", timeframe.ToString().EnumNameToTransportCase());
-            xmlWriter.WriteElementString("quantity", Quantity.AsString());
             xmlWriter.WriteStringIfValid("plan_code", PlanCode);
             xmlWriter.WriteIfCollectionHasAny("subscription_add_ons", AddOns);
-            xmlWriter.WriteStringIfValid("coupon_code", _couponCode);
+            xmlWriter.WriteStringIfValid("coupon_code", _couponCode);
+
+            // TODO change Quantity to int? in next version
+            if (Quantity > 0)

```

Comment:
I'd like to change Quantity to an optional instead of doing this but I'm going to do a follow up for that on the 1.5 branch.